docs/reference/ReleaseArtefacts.md

```diff
@@ -0,0 +1,121 @@
+# Release artefacts
```

Comment:
I wrote a related explanation: https://oboacademy.github.io/obook/reference/owl_formats_variants/

Maybe link together and reclassify the above as "explanation"?

Reply:
not this file here, this is clearly reference, but the one I linked I mean,